help/destinations/catalog/advertising/snap-inc.md

```diff
@@ -0,0 +1,126 @@
+
+4. You will be shown the Snapchat data that the Adobe Experience Platform will be be able to access. Click **Continue** to proceed with the connection process. 
+
+![Auth Screen 2](assets/auth2.png)
+
+After clicking continue, wait until you are redirected back to the Adobe Experience Platform. 
+
+### Fill in destination details {#destination-details}
+
+![Destination Details](assets/destinationdetails.png)
+
+To configure details for the destination, fill in the required fields and select **[!UICONTROL Next]**.
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Account ID]**: The Ad Account ID that is associated with the Ad Account that you will like to import your segments to. For more information about how to find this, please refer to [this documentation on the Snapchat Business Help Center](https://businesshelp.snapchat.com/s/article/biz-acct-id?language=en_US).
+
+>[!IMPORTANT]
+> 
+>Entering an incorrect or invalid Snapchat Ad Account ID will cause segment activation to fail. Please double check that you have entered the proper Ad Account ID.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate profiles and segments to streaming segment export destinations](../../ui/activate/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+## Validate data export {#exported-data}
+
+After activating segments to the *Snap Inc* destination, you will be able to see the segments in the Snap Ads Manager's [**Audiences** section](https://businesshelp.snapchat.com/s/article/audience-sharing). To navigate to this section, follow these steps:
+
+1. Log into the [Snap Ads Manager](https://ads.snapchat.com/)
+2. Select **Audiences** from the pulldown menu in the upper left corner of the screen. You will see the segments that you activated in the Adobe Experience Platform in the Audience Library:
+
+![Audiences](assets/audiences.png)
+
+Please note that when an Adobe segment is first activated to Snap Inc, you will initially see it as an empty audience. This is because it takes  time for the Adobe Experience Platform to build the segment and export the member data to Snap Inc. For more information about this, please refer to [Adobe's Segment Service documentation](https://experienceleague.adobe.com/docs/experience-platform/segmentation/home.html).
```

Comment:
Quick question about: 

"This is because it takes  time for the Adobe Experience Platform to build the segment and export the member data to Snap Inc. For more information about this, please refer to [Adobe's Segment Service documentation](https://experienceleague.adobe.com/docs/experience-platform/segmentation/home.html)."

I think it would be good if we could be just a bit more specific, because the page that we are linking to is pretty long. Where exactly in the page can users find more information about the time needed to build the segment and export member data?

Reply:
This behavior was pointed out to me by the Partner Integration Manager. I couldn't find any documentation that specifically mentions the time needed to build the segment, so I instead rewrote this passage to talk about how segment evaluation needs to be finished before data is exported and pointed the user to a document that talks about how segment evaluation works in Experience Platform. Please have a look at the most recent commit to see if this is okay.